test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -114,10 +118,14 @@ private static void generateFieldMethods(ClassWriter cw, String[] nameAndSigValu
 		generateGetter(cw, nameAndSigValue, className);
 		generateSetter(cw, nameAndSigValue, className);
 		generateWither(cw, nameAndSigValue, className);
+		generateSetterStatic(cw, nameAndSigValue, className);
```

Comment:
this function needs to check if it is a static field or not so that it only creates the correct type of field method

Reply:
to be clear, I should essentially just return early if it is not a static field?

Reply:
The function should be restructured such that there is a different path for static vs. instance fields

Reply:
Oh I thought your comment applied to changing the generateSetterStatic function but you're referring to the generateFieldMethods. I can make the change

Reply:
@tajila change has been made, pls review whenever you have a chance